wifiphisher/common/interfaces.py

```diff
@@ -319,233 +339,320 @@ def __init__(self):
         Setup the class with all the given arguments.
 
         :param self: A NetworkManager object
-        :param jamming_argument: The jamming argument given by user
-        :param ap_argument: The AP argument given by user
         :type self: NetworkManager
-        :type jamming_argument: str
-        :type ap_argument: str
         :return: None
         :rtype: None
-        .. seealso:: NetworkAdapter
         """
 
-        # Setup the fields
-        self._interfaces = {}
-        self.ap_iface = ""
-        self.jam_iface = ""
+        self._name_to_object = dict()
+        self._active = set()
```

Comment:
is the name `_name_to_object` clear enough to represent it is a dictionary doing mapping from adapter name to the `NetworkAdapter`?